glotaran/plugin_system/test/test_base_registry.py

```diff
@@ -330,22 +357,23 @@ def test_methods_differ_from_baseclass(
     """Inherited methods are the same as base class and overwritten ones differ"""
     result = methods_differ_from_baseclass(method_names, plugin, MockPlugin)
 
-    assert result == expected
+    assert list(result) == expected
```

Comment:
Why not change expected to also be a list?

Reply:
`expected` is a list, but `result` is a generator so we need to create a list from it to execute it.